rubyau/constitution.md

```diff
@@ -1,468 +1,779 @@
+  3. attempt in good faith to settle the dispute by mediation.
+
+2. The mediator must be—
+  1. a person chosen by agreement between the parties; or
   2. in the absence of agreement—
-    1. in the case of a dispute between a member and another member, a person appointed by the committee of the Association; or 
-    2. in the case of a dispute between a member and the Association, a person who is a mediator appointed or employed by the Dispute Settlement Centre of Victoria (Department of Justice). 
+    1. if the dispute is between a member and another member—a person appointed by the Committee; or
+    2. if the dispute is between a member and the Committee or the Association—a person appointed or employed by the Dispute Settlement Centre of Victoria.
+
+3. A mediator appointed by the Committee may be a member or former member of the Association but in any case must not be a person who—
+  1. has a personal interest in the dispute; or
+  2. is biased in favour of or against any party.
 
-5. A member of the Association can be a mediator. 
+#### 28 Mediation process
 
-6. The mediator cannot be a member who is a party to the dispute. 
+1. The mediator to the dispute, in conducting the mediation, must—
+  1. give each party every opportunity to be heard; and
+  2. allow due consideration by all parties of any written statement submitted by any party; and
+  3. ensure that natural justice is accorded to the parties throughout the mediation process.
 
-7. The parties to the dispute must, in good faith, attempt to settle the dispute by  mediation. 
+2. The mediator must not determine the dispute.
 
-8. The mediator, in conducting the mediation, must—
-  1. give the parties to the mediation process every opportunity to be heard; and 
-  2. allow due consideration by all parties of any written statement submitted by any  party; and 
-  3. ensure that natural justice is accorded to the parties to the dispute throughout  the mediation process. 
+#### 29 Failure to resolve dispute by mediation
 
-9. The mediator must not determine the dispute. 
+If the mediation process does not resolve the dispute, the parties may seek to resolve the dispute in accordance with the Act or otherwise at law.
 
-10. If the mediation process does not result in the dispute being resolved, the parties may  seek to resolve the dispute in accordance with the Act or otherwise at law. 
+## Part 4—General Meetings Of The Association
 
-## 10. Annual general meetings
+#### 30 Annual general meetings
 
-1. The committee may determine the date, time and place of the annual general meeting of the Association. 
+1. The Committee must convene an annual general meeting of the Association to be held within 5 months after the end of each financial year.
 
-2. The notice convening the annual general meeting must specify that the meeting is an annual general meeting. 
+2. The Committee may determine the date, time and place of the annual general meeting.
```

Comment:
In the original document, there is no subrule 2. The subrules are numbered 1, 3, 4 and 5.

Assuming a preexisting subrule 2 has been deleted, I have adjusted the numbering here to remove the gap.